velox/exec/AssignUniqueId.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+#include "velox/exec/Operator.h"
+#include "velox/vector/FlatVector.h"
+
+namespace facebook::velox::exec {
+
+class AssignUniqueId : public Operator {
+ public:
+  AssignUniqueId(
+      int32_t operatorId,
+      DriverCtx* driverCtx,
+      const std::shared_ptr<const core::AssignUniqueIdNode>& planNode,
+      int32_t uniqueTaskId,
+      std::shared_ptr<std::atomic_int64_t> rowIdPool);
+
+  bool isFilter() const override {
+    return false;
```

Comment:
The definition of this method is: // Returns true if 'this' never has more output rows than input rows.

Hence, it should return true.